src/protocols/secure_channel/tests/TestCASESession.cpp

```diff
@@ -334,22 +340,25 @@ CHIP_ERROR InitCredentialSets()
     return CHIP_NO_ERROR;
 }
 
-void TestContext::SetUpTestSuite()
+void TestCASESession::SetUpTestSuite()
 {
     CHIP_ERROR err = CHIP_NO_ERROR;
     LoopbackMessagingContext::SetUpTestSuite();
-    // TODO: use ASSERT_EQ, once transition to pw_unit_test is complete
-    VerifyOrDieWithMsg((err = chip::DeviceLayer::PlatformMgr().InitChipStack()) == CHIP_NO_ERROR, AppServer,
-                       "Init CHIP stack failed: %" CHIP_ERROR_FORMAT, err.Format());
-    VerifyOrDieWithMsg((err = InitFabricTable(gCommissionerFabrics, &gCommissionerStorageDelegate, /* opKeyStore = */ nullptr,
-                                              &gCommissionerOpCertStore)) == CHIP_NO_ERROR,
-                       AppServer, "InitFabricTable failed: %" CHIP_ERROR_FORMAT, err.Format());
-    VerifyOrDieWithMsg((err = InitCredentialSets()) == CHIP_NO_ERROR, AppServer, "InitCredentialSets failed: %" CHIP_ERROR_FORMAT,
-                       err.Format());
+
+    err = chip::DeviceLayer::PlatformMgr().InitChipStack();
+    ASSERT_EQ(err, CHIP_NO_ERROR) << "Init CHIP stack failed: " << std::hex << err.Format();
```

Comment:
we can drop the extra messages bits ... I think if we include StringBuilderAdapters (after https://github.com/project-chip/connectedhomeip/pull/33505) we can do:

```cpp
ASSERT_EQ(chip::DeviceLayer::PlatformMgr().InitChipStack(), CHIP_NO_ERROR);
```

And you will get a nicely formatter info on failure. The only purpose of these messages seem to have been for having the CHIP-error nicely formatted and our macros can do that already.

Reply:
Also, one thing to add. Generally it's always better to do:
```c++
ASSERT_EQ(chip::DeviceLayer::PlatformMgr().InitChipStack(), CHIP_NO_ERROR);
```
instead of:
```c++
CHIP_ERROR err = chip::DeviceLayer::PlatformMgr().InitChipStack();
ASSERT_EQ(err, CHIP_NO_ERROR);
```
In the former case, the test framework will be able to print the name of the function which has failed instead of a variable name, which might not say much (then you will have to look at the test source code to know which function has failed).

Reply:
Fixed